src/QsCompiler/DataStructures/DataTypes.fs

```diff
@@ -53,6 +53,14 @@ type QsNullable<'T> = // to avoid having to include the F# core in the C# part o
         | Some v -> Value v
         | None -> Null
 
+/// Factories for <see cref="QsNullable"/>.
+module QsNullable =
```

Comment:
Is this specifically only to mark the string for the execution target as nullable? Since we already have NonNullable, I'd suggest to drop this.

Reply:
I removed it.